frontend/src/components/common/shareAccessModal/index.tsx

```diff
@@ -103,35 +122,35 @@ export const ShareAccessModal: React.FC<ShareAccessProp> = props => {
           ? {
               value,
               label: (
-                <>
+                <Tooltip title={label} placement="left">
```

Comment:
Not use tooltip in here, Use <Typography.Text ellipese={{tooltip: true}}>{content}</Typography.Text> instead